events/imperia_dead_man_events.txt

```diff
@@ -160,6 +160,15 @@ dead_man.2 = {
 		add_loyalists = {
 			value = 0.35
 		}
+		if = {
+			ig:ig_intelligentsia = {
+				limit = {
```

Comment:
You cannot go directly from an if into the intelligentsia here, you need to either invoke intelligentsia both witin the limit and below the limit, in the effect, or possibly invoke the intelligentsia outside the if.

Reply:
Basically this